src/ansible/roles/common/tasks/main.yml

```diff
@@ -65,9 +65,19 @@
     mode: '0600'
   with_items:
   - { src: 'root.id_rsa', dest: 'id_rsa' }
-  - { src: 'root.id_rsa.pub', dest: 'authorized_keys' }
+#  - { src: 'root.id_rsa.pub', dest: 'authorized_keys' }
```

Comment:
If you aren't going to use this line then it's better to remove it.

Reply:
Deleted.